explainer.md

```diff
@@ -1,751 +1,423 @@
+
+The `::page-transition` is rendered in a top-level, filling the viewport.
+
+Once this is in place, rendering is resumed.
+
+# Default styles & animation
+
+## `::page-transition-container(*)`
 
-## Part 3: The transition
+Default styles:
 
-Everything is now in place to perform the transition. The developer can animate
-the transition elements created by the UA using the usual APIs, such as CSS and
-web animations.
+- Absolutely positioned to the top left of the parent.
+- Width and height of the outgoing element.
+- A transform that places it in the viewport position of the outgoing element.
 
-https://user-images.githubusercontent.com/93594/141100217-ba1fa157-cd79-4a9d-b3b4-67484d3c7dbf.mp4
+Default animation:
 
-Note that the browser defers displaying elements from Page-B and starting the
-animation until Page-B is ready for first render. This is currently driven by
-internal browser heuristics and is being standardized in the proposal
-[here](https://github.com/whatwg/html/issues/7131).
+- Width and height animates to the dimensions of the incoming element.
+- Transform animates to a transform that places it in the viewport position of the incoming element.
 
-## Part 4: The end
+## `::page-transition-image-wrapper(*)`
 
-When the transition is complete, the transition elements created by the UA are
-removed, revealing the real Page-B. The transition completes once no pseudo
-element has an active animation.
+Default styles:
 
-# The cross-document transition API
+- Absolutely positioned with 0 inset.
+- `isolation: isolate` to aid with cross-fading.
 
-## Page-A
+Default animation: none.
 
-Page-A must offer elements to use for a transition, otherwise no transition will
-happen.
+## `::page-transition-outgoing-image(*)`
 
-### Via CSS
+Default styles:
 
-Page-A can offer an element to be used in a transition via CSS, using the
-`page-transition-tag` property, with a default value of `none`:
+- A replaced element displaying the capture of the outgoing element, with a natural aspect ratio of the outgoing element.
+- Absolutely positioned to the inline & block start.
+- `mix-blend-mode: plus-lighter` to allow for a true cross-fade.
+- 100% block size
+- Auto inline size
+- The captured `object-view-box`
+
+The `object-view-box` allows the image to be the layout size of the element, but allow overflow (to accommodate ink-overflow) and underflow (cropping to save memory) in the image data.
+
+Default animation:
+
+- Opacity animates from 1 to 0.
+
+Note that the height of this element is auto, so it won't stretch the image as the container changes height. The developer can change this if they wish.
+
+## `::page-transition-incoming-image(*)`
+
+As above, but animates from opacity 0 to 1.
+
+# Ending the transition
+
+The transition ends once a state is reached where no animations are running on any of the transition pseuduo-elements.
+
+Once this happens, the pseudo-elements are removed, and the real DOM underneath is shown.
```

Comment:
We should clarify somewhere before this that the tagged DOM elements are not painted in the real DOM while they are being displayed via pseudo-elements during the transition. The effect is similar to the elements and their 
descendants having `visibility: hidden` and `pointer-events: none`.